airflow/www/views.py

```diff
@@ -3953,71 +3955,92 @@ def __getattribute__(self, attr):
                 return action_logging(event=f"{self.route_base.strip('/')}.{permission_str}")(attribute)
         return attribute
 
+    @expose("/show/<pk>", methods=["GET"])
```

Comment:
Shortly we will vendor-in the whole FAB.. I just wonder what's left in FAB :) ?

Reply:
Target: NOTHING :D Joking aside, I think at some point it would be ideal to replace these auto generated views with some using the new UI (React + Rest API as backend)

Reply:
cc: @bbovenzi  @pierrejeambrun  ^^